problem_builder/public/css/problem-builder.css

```diff
@@ -118,16 +118,62 @@
 
 .mentoring .checkmark-incorrect {
     font-size: 22pt;
-    color: #ff0000;
+    color: #c1373f;
```

Comment:
nit: where does this color come from?  Does it match an edX standard color somewhere?

Reply:
I just moved it from lms.css here. I don't see it used in edX, so not sure why this was picked.

Reply:
Ah ok, understood.  👍